.editorconfig

```diff
@@ -53,4 +53,9 @@ indent_size = 2
 
 # Makefile
 [Makefile]
-indent_style = tab
\ No newline at end of file
+indent_style = tab
+
+# Kconfig
+[Kconfig]
```

Comment:
While Kconfig is typically tabs there are situations where some indentation is being done with spaces i.e. for help descriptions.